plutarch-test/goldens/api.map.uplc.eval.golden

```diff
@@ -0,0 +1,19 @@
+lookup (program 1.0.0 True)
+lookupData (program 1.0.0 True)
```

Comment:
Hence [this advice](https://github.com/Plutonomicon/plutarch/pull/396/files#r845580960), because otherwise `eval.golden` will be filled with `True`, which is not very useful.